.github/workflows/persubmit.yml

```
@@ -58,7 +58,7 @@ jobs:
           ti test -vr2 -t2
 
   build_and_test_cuda:
-    name: Build and Test (CUDA)
+    name: Build and Test (CUDA and OpenGL)
```

```suggestion
    name: Build and Test (GPU)
```
We may add OpenCL to zhen later too.